integration/spark/src/test/common/java/io/openlineage/spark/agent/SparkContainerIntegrationTest.java

```diff
@@ -244,10 +244,9 @@ public void testCreateAsSelectAndLoad() throws IOException, InterruptedException
     String startCTASEvent = new String(readAllBytes(eventFolder.resolve("pysparkCTASStart.json")));
     String completeCTASEvent = new String(readAllBytes(eventFolder.resolve("pysparkCTASEnd.json")));
 
-    //    String startLoadEvent =
-    //      new String(readAllBytes(eventFolder.resolve("pysparkLoadStart.json")));
-    //    String completeLoadEvent =
-    //      new String(readAllBytes(eventFolder.resolve("pysparkLoadComplete.json")));
```

Comment:
Can we just delete these instead of commenting them out?

Reply:
You've commented on a line that was changed to uncommented :slightly_smiling_face:

Reply:
🤦🏽‍♂️